all.json

```
@@ -1896,6 +1923,11 @@
     "walletconnect-supports.org",
     "walletconnect-sync.org",
     "walletconnect-validate.org",
+    "walletconnect.com.finance-airdrop.info",
+    "walletconnect.com.holders-airdrop.info",
+    "walletconnect.finance-airdrop.info",
+    "walletconnect.holders-airdrop.info",
```

```suggestion
    "walletconnect.finance-airdrop.info",
    "walletconnect.holders-airdrop.info",
    "finance-airdrop.info",
    "holders-airdrop.info",
```

Blocking the top-level.